W 11239241 internet browser change cp (#331)
mulesoft/docs-studio: modules/ROOT/pages/index.adoc

== Requirements

> * Studio 6.x will miss some features in Windows due to the fact that Anypoint Platform no longer supports Internet Explorer (see https://docs.mulesoft.com/general/browser-support):

Reviewer: I think that the practice in docs is not to put the direct link but a text with an internal link that it is resolved when docs are built. If it is not possible for some reason then leave a linked text for that URL.

Author: You're right!
Thanks for the reminder! ;)

> ** Anypoint Studio will not display the Anypoint Platform views if your system has a default Internet browser.

Reviewer: this line must be removed... causes confusion... It gives false sense that there is a workaround and there isn't really.

One thing that maybe could be added is that the below bullets are the currently ones that does not work and maybe in the future there will be others when more services start to remove support to IE.

Author: Thanks for the explanations! ;)

== Requirements

> * Studio 6.x will miss some features in Windows due to the fact that Anypoint Platform no longer supports Internet Explorer (see xref:docs-general::browser-support.adoc[Browser Support for Anypoint Platform]).

Reviewer 2: ```suggestion
* Studio 6.x doesn't use some features in Windows because Anypoint Platform no longer supports Internet Explorer (see xref:docs-general::browser-support.adoc[Browser Support for Anypoint Platform]).
```

** Internet Explorer version 11 and earlier are not supported.

> ** Anypoint Studio will not display the Anypoint Platform views if your system has a default Internet browser.

Reviewer: I would change this by something like:

> ... features in Windows due to Internet Explorer is not supported anymore by Anypoint Platform (See https://docs.mulesoft.com/general/browser-support):

and remove these last 2 sub-items because:
- Not a single one IE version is supported anymore
- it gives the sensation that there is a workaround configuring other browser by default, which is not, it depends on the native browser of Windows. 

Maybe it could be added something about that some Web UI parts could be still working but in the future could not. This will depend on the development of those parts of the web services on the Anypoint Platform.

Author: I've made some changes, but I'm not sure I understood your comments. Please, can you take a look at it again? Thanks!

Author: Thanks for the comments!
I've made some changes based on your comments.
If some parts of the web UI will not work in the future, it does not seem efficient to comment about it now.
Please, can you take a look at them and let me know if they are OK? Thanks!

> ** Anypoint Studio will not display the Anypoint Platform views if your system has a default Internet browser.

Reviewer: I would change this by something like:

> ... features in Windows due to Internet Explorer is not supported anymore by Anypoint Platform (See https://docs.mulesoft.com/general/browser-support):

and remove these last 2 sub-items because:
- Not a single one IE version is supported anymore
- it gives the sensation that there is a workaround configuring other browser by default, which is not, it depends on the native browser of Windows. 
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -

Author: I've made some changes, but I'm not sure I understood your comments. Please, can you take a look at it again? Thanks!

Author: Thanks for the comments!
I've made some changes based on your comments.
If some parts of the web UI will not work in the future, it does not seem efficient to comment about it now.
Please, can you take a look at them and let me know if they are OK? Thanks!

> ** Authentication over MFA.

Reviewer 2: ```suggestion
** Authenticate with MFA.
```

Reviewer 2: (To make it parallel with the other bullets.)

** Deploy to CloudHub.

> As IE is no longer supported, other features may not work in the future.

Reviewer 2: ```suggestion
Because Internet Explorer is no longer supported, other features might not work in the future.
```

Reviewer 2: (may implies permission; use might for probability)

Author: Like it! Thanks!

** Other supported runtime versions work fine with both Java 7 or 8.

* Studio 6 is not compatible with Mule 4.

> * From Studio 6 on, in Windows, you cannot use Exchange UI or deploy Mule apps to CloudHub.

Reviewer: I would not duplicate the info here... if you want to add something maybe I would add a reference to the previous paragraph.

Reviewer: also, it really isn't from Studio 6 that it is not supported... it is since IE is not supported anymore on the Platform as previously commented... this doesn't mean that from one day to another all the platform doesn't work... for example there are parts that still work but some services like Exchange doesn't allow anymore to access with IE.

Author: Good catch on your first comment!
Regarding IE, tough some parts may work, if we are stating that IE is not compatible with our platform, we are not responsible for the outcome of using our platform with IE, i.e.: it may work, it may not work, or it may render an unexpected result.
Thanks!

> * From Studio 6 on, in Windows, you cannot use Exchange UI or deploy Mule apps to CloudHub.

Reviewer: I would not duplicate the info here... if you want to add something maybe I would add a reference to the previous paragraph.

Author: Good catch!
Thanks!
